src/Microsoft.CodeAnalysis.Analyzers/Core/CodeAnalysisDiagnosticsResources.resx

```diff
@@ -529,4 +529,16 @@
   <data name="PreferIsKindFix" xml:space="preserve">
     <value>Use 'IsKind' instead of 'Kind'</value>
   </data>
+  <data name="MissingSourceGeneratorAttributeDescription" xml:space="preserve">
+    <value>'{0}' implements 'ISourceGenerator' but does not have the 'Generator' attribute.</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeMessage" xml:space="preserve">
+    <value>Missing 'Generator' attribute</value>
+  </data>
+  <data name="MissingSourceGeneratorAttributeTitle" xml:space="preserve">
+    <value>Missing 'Generator' Attribute</value>
+  </data>
+  <data name="AddGeneratorAttribute" xml:space="preserve">
+    <value>Add 'Generator' Attribute</value>
```

Comment:
Consider making attribute lowercase.

Reply:
Sure, changing to `Apply Generator attribute for '{0}'.`